packages/party_walls/tests/test_party_walls.py

```diff
@@ -1,89 +1,82 @@
-import pytest
 import os
-from dagster import build_asset_context, materialize_to_memory, RunConfig
-from bag3d.party_walls.assets.party_walls import distribution_tiles_files_index, \
-    party_walls_nl, cityjsonfeatures_with_party_walls_nl, \
-    features_file_index_generator
-from bag3d.common.resources.database import db_connection, container
-from bag3d.common.resources.files import file_store
+
 import pandas as pd
+import pytest
+from bag3d.party_walls.assets.party_walls import (
+    TilesFilesIndex, cityjsonfeatures_with_party_walls_nl,
+    distribution_tiles_files_index, features_file_index,
+    features_file_index_generator, party_walls_nl)
+
+TILE_IDS = ('10/564/624', '10/564/626', '10/566/624', '10/566/626', '9/560/624')
+
+import pickle
+from pathlib import Path
 
+from dagster import asset
 
-@pytest.fixture(scope="session")
-def resource_container():
-    return container.configured({"id": "testid"})
 
+@asset(name="party_walls_nl")
+def mock_party_walls_nl(output_data_dir)  -> pd.DataFrame:
+    return pd.read_csv(output_data_dir / "party_walls_nl.csv")
 
-@pytest.fixture(scope="function")
-def resource_db_connection():
-    """db_connection resource"""
-    return db_connection.configured({
-        "host": "localhost",
-        "port": 5432,
-        "user": os.environ.get("PYTEST_DB_USER"),
-        "password": os.environ.get("PYTEST_DB_PW"),
-        "dbname": "baseregisters_test"
-    })
 
+@asset(name="distribution_tiles_files_index")
+def mock_distribution_tiles_files_index(output_data_dir)  -> TilesFilesIndex:
+    return pickle.load(open(output_data_dir / "distribution_tiles_files_index.pkl", "rb"))
 
-@pytest.fixture(scope="session")
-def resource_file_store(output_data_dir):
-    """file_store resource pointing to the test output location"""
-    return file_store.configured({"data_dir": str(output_data_dir), })
 
+@asset(name="features_file_index")
+def mock_features_file_index(output_data_dir)  -> dict[str, Path]:
+    return pickle.load(open(output_data_dir / "features_file_index.pkl", "rb"))
 
-def test_distribution_tiles_files_index(export_dir_uncompressed):
-    """Can we parse the CityJSON tiles and return valid data?
-    Currently hardcoded for Balázs' local test data.
-    """
-    tile_ids = ('10/564/624', '10/564/626', '10/566/624', '10/566/626', '9/560/624')
 
+def test_distribution_tiles_files_index(context):
```

Comment:
It seems to me that this test basically loads some data from a picke and verifies that the data is correct, but it does not run any pipeline code. Is that correct?
In that case, I think it could be removed, because it just adds noise to the tests.
Or if the pickle needs to be verified, maybe move the verification to a fixture with a "session" scope?

Reply:
I think this tests the function distribution_tiles_files_index, but you probably mean  the test test_features_file_index_generator